neural_modelling/Doxyfile

```diff
@@ -907,7 +907,7 @@ USE_MDFILE_AS_MAINPAGE =
 # also VERBATIM_HEADERS is set to NO.
 # The default value is: NO.
 
-SOURCE_BROWSER         = NO
+SOURCE_BROWSER         = YES
```

Comment:
Generates a link to the (pretty-printed) source for all files, not just headers.